javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,39 @@
+let currencyRates = {
+  "timestamp": Date.now(),
+  "base": "EUR",  
+  "date": new Date().toISOString().split('T')[0], 
+  "rates": {
+      "EUR": 1 
+  }
+};
+
+function addCurrencyRate(currency, rate){
+  if (!currencyRates.rates[currency]) {
```

Comment:
you can also check currency and rate input are not undefined or empty before adding the currency to the list.